### ISSUE TITLE
Reload/QuickFix library does not work if library is logging to console on init

### ISSUE THREAD

**Issue by Mario-P** [nokia/RED#191]
If library has logging to console enabled in init - logger.info("Importing library", also_console=True). Reloading or QuickFix of such library causes error (see attached). 
Auto discovery of such library works fine.

Project - [RED_Tests.zip](https://github.com/nokia/RED/files/1780043/RED_Tests.zip)

Error:
![logger_error](https://user-images.githubusercontent.com/22638505/36964687-ba15423c-2057-11e8-9d93-a5f73f1115b8.JPG)

**Comment by KrzysztofJozefowicz** [comment]
this is only visible when when library file is changed and RED reloads the lib or clicking reload in Referenced Libraries.
Root cause is adding newline to sys stdout 
```
  def __init__(self):
        import sys
        sys.__stdout__.write('\n\r')
```

**Comment by KrzysztofJozefowicz** [comment]
released in 0.8.4
